src/cogs/modmail.py

```diff
@@ -60,11 +60,12 @@ async def select_type(self, interaction, selection):
 
     @discord.ui.button(label = "Create ticket", custom_id = "persistent_view:create_ticket")
     async def button_callback(self, interaction, button):
-        if not self.children[0].values:
-            await interaction.response.send_message("Please select an option", ephemeral = True)
-            return
+        for child in self.children:
```

Comment:
can there be more than one child?

Reply:
Yes, the button can be too, im thinking maybe something gets messed up and button is the 0th child

Reply:
gotcha